rust/arrow/src/array.rs

```diff
@@ -800,6 +1082,17 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_timestamp_fmt_debug() {
+        // TODO might be preferable to format these as datetime strings
```

Comment:
Yes - I agree, datetime strings would be better.

Reply:
Found a way to do it with specialisation